nautiluszim/scraper.py

```diff
@@ -332,6 +336,57 @@ def extract_to_fs(
                     return
                 raise exc
 
+    def check_archiveless_collection(self):
+        with open(self.collection, "r") as fp:
```

Comment:
Instead of this we can call `test_collection` beforehand as this is basically a duplicate.
It should probably be renamed. Maybe `load_collection` ?